src/protocols/aeternity/views/TransferSendModal.vue

```diff
@@ -90,6 +95,13 @@ export default defineComponent({
         && isUrlValid(transferData.value.address)
     ));
 
+    const isPrimaryButtonDisabled = computed(() => (
+      error.value
+      || !isAeNodeReady.value
+      || !transferData.value.address
+      || !transferData.value.amount
```

Comment:
```suggestion
      !error.value
      && isAeNodeReady.value
      && transferData.value.address
      && transferData.value.amount
```

this is only personal suggestion. I think, if we have a lot of negation, then code is more not readable.

Reply:
It's a matter of readability. When you convert the code into sentence it makes more sense with negations: "the button is disabled if there is an error or the node is not ready, or address is empty or there is no amount". If we invert it it will take more time to understand.

Reply:
like i said, this is individual. For me is ok

Reply:
OK, resolving.